app/views/set_crystals/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with url: "/crystal_sets/#{@crystal_set.id}/crystals", method: :post, local: true do |form| %>
+  <%= form.label :name, "Name" %>
+  <%= form.text_field :name %>
```

Comment:
if you add `:required => true' inside the form will make is so the field has to have something in it to pass on.

Reply:
👀 oooh lala!! I will incorporate this